src/test/java/by/exadel/internship/userTest/UserTest.java

```diff
@@ -1,23 +1,33 @@
 package by.exadel.internship.userTest;
 
 import by.exadel.internship.InternshipApplicationTests;
-import by.exadel.internship.dto.user.UserDTO;
+
 import by.exadel.internship.dto.enums.UserRole;
+import by.exadel.internship.dto.user.UserDTO;
+import by.exadel.internship.entity.User;
+import by.exadel.internship.exception_handing.NotFoundException;
+import by.exadel.internship.repository.UserRepository;
 import com.fasterxml.jackson.core.type.TypeReference;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpMethod;
+import org.springframework.test.context.jdbc.Sql;
 import org.springframework.test.web.servlet.MvcResult;
 import org.springframework.web.util.UriComponentsBuilder;
 
 import java.net.URI;
 import java.util.List;
+import java.util.UUID;
 
-import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.*;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
-
+@Sql(scripts = "/user-test-data.sql")
```

Comment:
If your test requires some specific test data, you need to clean up DB after this tests.

Reply:
Done. Added test insert and delete data for each test